projects/angular/src/data/datagrid/built-in/filters/datagrid-numeric-filter.spec.ts

```diff
@@ -127,8 +128,7 @@ export default function (): void {
       const input: HTMLInputElement = document.querySelector("input[type='number']");
       spyOn(input, 'focus');
       expect(input.focus).not.toHaveBeenCalled();
-      // The `requestAnimationFrame` is mocked through `setTimeout(fn, 16)`.
-      tick(16);
+      animationFrameTick();
```

Comment:
The animation frame changes are unrelated, so these changes should be in a separate commit.

Reply:
That's a good practice that I full heartedly support.
Especially when chore-type of changes in hundreds of files are intermixed with meaningful code-changes.
I'll keep it in the same PR, but add a new commit.

Reply:
done

Reply:
Thanks.